keyform/templates/keyform/add_form.html

```diff
@@ -25,7 +25,7 @@
                         {{ form.reason_for_request }}
                     </div>
                     <div class="large-6 columns">
-                        <label>{% trans 'Filed by:' %}</label>
+                        <label>{% trans 'Submitted by:' %}</label>

```

Comment:
If we're changing this, we should change it on the request.html and email templates as well.